DRAFT_RELEASE_NOTES.md

```diff
@@ -101,4 +101,5 @@ If you added any custom configurations to the `sparkoperator` service account pe
 
 # What's Changed
 - `pyproject.toml` files updated to allow for Python version `>=3.8`.
-- All SageMaker modules and corresponding references were removed in commit bdcbb409.
+- All SageMaker modules and corresponding references were removed in commit 8ce3f39.
+- The Service Discovery module and corresponding references were removed in commit 4a1d7abf. 
```

Comment:
You currently have 2 commits, so this is probably gonna change if you squash them

Reply:
Unfortunately I need to have two commits to do this. When you squash them, git changes the commit hash.

Reply:
Oh that makes sense, can't predict the squash commit hash